src/test/options-test.js

```diff
@@ -617,3 +617,38 @@ group("the existing modifier", () => {
   existingSuite.run(lab, { OPTIONS: { alwaysTransform: false, alwaysSet: true } });
 
 });
+
+group("the failureTransform as a option for default use", () => {
+  lab.test("should use the global failure transformed if its defined and failure transform is not provided as part of the mapping", done => {
+    const source = {
```

Comment:
This test data appears to be irrelevant? I'm not sure if you meant to test fooNull and forgot or whether it is just noise.